File: eCakeShop.Models/Recenzija.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace eCakeShop.Models
{
    public class Recenzija
    {
        public int RecenzijaID { get; set; }
        public string SadrzajRecenzije { get; set; }
        public int Ocjena { get; set; }
        public DateTime DatumKreiranja { get; set; }
        public int KorisnikID { get; set; }
        public virtual Korisnik Korisnik { get; set; }
    }
}

File: e_cakeshop_mobile/lib/models/recenzija.dart

import 'package:e_cakeshop_mobile/models/korisnik.dart';
import 'package:json_annotation/json_annotation.dart';
part 'recenzija.g.dart';

@JsonSerializable()
class Recenzija {
  int? recenzijaID;
  String? sadrzajRecenzije;
  int? ocjena;
  DateTime? datumKreiranja;
  int? korisnikID;
  Korisnik? korisnik;

  Recenzija();
  factory Recenzija.fromJson(Map<String, dynamic> json) =>
      _$RecenzijaFromJson(json);

  Map<String, dynamic> toJson() => _$RecenzijaToJson(this);
}
